src/main/java/com/epam/poc/pageUIs/HeaderPageUI.java

```diff
@@ -1,6 +1,13 @@
 package com.epam.poc.pageUIs;
 
+import org.openqa.selenium.By;
+
 public class HeaderPageUI {
 
-    public static final String JOIN_AS_SELLER_HYPERLINK_CSS = "a[href='https://shopee.vn/m/sell-on-shopee']";
+    public static final By followUsOnFacebookLink = By.cssSelector("[title*='Facebook']");
```

Comment:
Hi Nhi, you should rename these variables with an uppercase letter.
Ex: "followUsOnFacebookLink " => "FOLLOW_US_ON_FACEBOOK_LINK_BY"